Cargo.toml

```
@@ -38,6 +39,7 @@ esplora-reqwest = ["esplora", "bdk/use-esplora-reqwest"]
 compiler = ["bdk/compiler"]
 compact_filters = ["bdk/compact_filters"]
 rpc = ["bdk/rpc"]
+reserves = ["bdk/electrum", "bdk-reserves"]
```

if you add the reserve feature like this, then inside the lib you don't have to specify both `reserve` and `electrum` feature gates. Only `reserve` would work as it would automatically open `electrum` stuffs.  
```suggestion
reserves = ["electrum", "bdk-reserves"]
```

---

Actually, reserves should not automatically pull in electrum. I missed this part in the recent separation.

---

Noted..